auth_otp/services/otp_service.py

```
@@ -0,0 +1,341 @@
+                otp_request.save()
+                
+                # افزایش تلاش ناموفق
+                self._add_failed_attempt(phone_number)
+                
+                return False, {
+                    'error': 'send_failed',
+                    'message': result.get('error', 'خطا در ارسال کد تأیید')
+                }
+                
+        except Exception as e:
+            logger.error(f"Error in send_otp: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)